k8s/helm/templates/job.yaml

```diff
@@ -10,11 +10,11 @@ spec:
       restartPolicy: Never
       containers:
       - name: {{ include "warp.fullname" . }}
-        image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
+        image: "{{ .Values.image.repository }}:{{ include "warp.imageVersion" . }}"
```

Comment:
are you trying to propose a breaking change?

Reply:
Sorry for the (heavily) delayed reply. This shouldn't be a breaking change, as the previous logic would only ever default the image tag to `latest`, since that's the hard coded `appVersion` in Chart.yaml. My change here will still default to this behaviour, but if you pass `image.version` in your values file you can now specify a specific version of the Warp image to use with the installation.